core/hydromet_JSON_to_DSS.py

```diff
@@ -3,8 +3,8 @@
 
 #---------------------------------------------------------------------------#
 def main(forcing_dir: plib, outputs_dir: plib, bin_dir: plib, filename: str, 
-                variable: str="Excess-Rainfall", data_type: str='INST-VAL', 
-                            units: str='INCHES', remove_temp_files: bool=True, 
+                jsonname: str, variable: str="Excess-Rainfall", data_type: str='INST-VAL', 
```

Comment:
Need to specify a default value for jsonname or it will break JSON_to_DSS.ipynb. Replace "jsonname: str" with "jsonname: str = ' '  "